### ISSUE TITLE
[1.14.4] Client crash on creating a new world.

### ISSUE THREAD

**Issue by ProsperCraft** [Xalcon/TorchMaster#97]
https://gist.github.com/ProsperCraft/6e54f7740782a2b2c77999f463277d77

torchmaster-2.0.0.6-alpha.jar
forge-1.14.4-28.0.49

**Comment by Xalcon** [comment]
Looks like a MineColonies bug to me. Forge crashes because mine colonies returns null from a method that must not return null:
```
java.lang.RuntimeException: Provider com.minecolonies.coremod.event.capabilityproviders.MinecoloniesWorldColonyManagerCapabilityProvider.getCapability() returned null; return LazyOptional.empty() instead!
```

**Comment by Xalcon** [comment]
Not sure where you got the minecolonies 1.14 version. It got fixed in their v1.14 source, see
https://github.com/ldtteam/minecolonies/commit/ecf67571ad8c2f317cd038b2d2a20170b29e17e9

**Comment by ProsperCraft** [comment]
I am a Patreon, and it is a Patreon only release.

Thank you for looking in to it.
